aten/src/ATen/native/mkldnn/BinaryOps.cpp

```diff
@@ -63,6 +75,38 @@ Tensor& mkldnn_add_(Tensor& self, const Tensor& other, Scalar alpha) {
   return native::mkldnn_add_out(self, self, other, alpha);
 }
 
+Tensor& mkldnn_mul_out(Tensor& result, const Tensor& self, const Tensor& other) {
+  AT_ASSERTM(result.sizes() == self.sizes(),
+             "mkldnn_mul_out: currently mkldnn not support broadcasting");
```

Comment:
If it's about broadcasting, you probably should check `self.sizes() == other.sizes()`?

Reply:
Perharps the warning message should be "mkldnn_mul_out: the output size should be same as input size"? I want to make sure the output size should be same as input, the broadcasting checking in Line-92. Thanks!

Reply:
Yeah makes senses